lib/commands/connect/shared.js

```diff
@@ -182,6 +182,19 @@ exports.withConnection = function(context, heroku) {
   });
 }
 
+var withMapping = exports.withMapping = function(connection, object_name) {
+  return Q.Promise(function(resolve, reject) {
+    var found = false, object_name_lower = object_name.toLowerCase();
```

Comment:
Hrm, it does seem we're using separate lines pretty much everywhere else. Easy fix, and I'll be more careful of it going forward.